CFReader.py

```diff
@@ -17,13 +17,14 @@ def get_downloads(project):
 
 
 def get_versions(project):
-    response = get_project(project)
-    pattern = '<h4 class="e-sidebar-subheader overflow-tip">\s+<a.*?>\s+Minecraft (.*?)\s+</a>\s+</h4>'
-    versions = re.findall(pattern, response)
-    if len(versions) > 0:
-        return versions
+    r = urllib2.urlopen('https://minecraft.curseforge.com/projects/' + project + '/files')
```

Comment:
Could you change this back to the `get_project` function. Just to keep the fetching in one place.
Althought it would be something like `get_project(project + '/files')`.  Maybe add a `get_project_files` function? I'll leave that to choice to you.

Reply:
Sure, I had that originally but changed it once I started getting frustrated >.>